tremor-script/src/ast.rs

```diff
@@ -665,6 +689,10 @@ where
 /// A tremor script instance
 #[derive(Clone, Debug, PartialEq, Serialize)]
 pub struct Script<'script> {
+    /// Start
+    pub start: crate::pos::Location,
```

Comment:
for consistency in AST we should probably follow the same `mid` (metadata id) logic we use for all the other AST elements instead of inlining the start/end

Reply:
Well spotted - yes it really should have a mid 👍

Reply:
Done - added a todo for making the name name captured as metadata capture better context in future - hardwired it for expedience now.